### DISCUSSION TITLE
Organize snapshot in models

### DISCUSSION THREAD

**Discussion by mattiaz9** [directus/directus#12179] [Feature Requests]
Right now a single snapshot file is hard to read and edit, in the case I'd want to manually make some changes.

My idea is to organize the snapshot in a subfolder, something like this:
```
/snapshot
  /collection-a
    /collection.yaml
    /fields.yaml
    /relations.yaml
  /collection-b
    /...
  /...
```

This would make it much easier to find fields and relations related to a collection.

**Comment by azrikahar** [comment]
That's be a great enhancement indeed! Just to tie them in for discoverability, there are also similar discussions such as #11262, #11249 and #9214.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
